### PR TITLE
docs: README change of file path for symbolic linking pre-commit under hooks

### PR DESCRIPTION
**Pull request by uchukwu**
README change of file path for symbolic linking of pre-commit under hooks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ asdf install
 mix deps.get
 
 # add pre-commit hook to verify formatting/tests/types
-ln -s ../../hooks/pre-commit .git/hooks/pre-commit
+ln -s ./hooks/pre-commit .git/hooks/pre-commit
```
Comment (bklebe) [inline README.md:26]:
```suggestion
ln -s hooks/pre-commit .git/hooks/pre-commit
```

**Review by bklebe — APPROVED** [review]

**Review by paulswartz — CHANGES_REQUESTED** [review]
@bklebe @uchukwu does doing the link this way run the tests for you? It doesn't on my machine.

**Comment by bklebe** [comment]
> @bklebe @uchukwu does doing the link this way run the tests for you? It doesn't on my machine.

I should have tested this, it doesn't seem to work. I approved it because it looked right from where the hooks file is, but I assume it's because the symbolic link needs to be relative to the .git folder.

**Comment by bklebe** [comment]
As an alternative you could use the absolute path to make this instruction less confusing, i.e. `ln -s "$(pwd)/hooks/pre-commit .git/hooks/pre-commit`. Or it looks like you can set `git config core.hooksPath hooks/` from the root of the working tree as well.

**Review by bklebe — CHANGES_REQUESTED** [review]
requires changes, per @paulswartz

**Pull request closed without merging**